.travis.yml

```diff
@@ -6,12 +6,22 @@ cache:
   directories:
     - ${HOME}/.opam
     - facebook-clang-plugins
+addons:
+  homebrew:
+    packages:
+      - gmp
+      - mpfr
+      - sqlite
+      - lzlib
 before_install:
-  - wget -O ${HOME}/opam https://github.com/ocaml/opam/releases/download/2.0.3/opam-2.0.3-x86_64-linux
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then ARCH="darwin"; else ARCH="linux"; fi
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then export PATH="/usr/bin:${PATH}"; fi
+  - wget -O ${HOME}/opam https://github.com/ocaml/opam/releases/download/2.0.3/opam-2.0.3-x86_64-${ARCH}
   - chmod +x ${HOME}/opam
   - export PATH=${HOME}:${PATH}
   - export OPAMYES=1
   - export OPAMJOBS=2
+  - rm -rf ${HOME}/.opam
```

Comment:
Note that there is hope that Travis caches the opam dir between runs (at least that was the motivation in #1075), so we do not want to force a deletion of the directory here (but maybe that was just for your test?).

Reply:
Yup that was just for my test - I recall there being something odd about installation of some of the libraries against homebrew vs system paths when I was making the homebrew formula, so I just wanted to make sure that wasn't screwing anything up